processing.py

```diff
@@ -0,0 +1,109 @@
+# processing Carvana image dataset
+# All input data should be kept in a folder called 'input'
+
+import skimage
+from skimage import data
+from scipy import ndimage
+import numpy as np
+import pandas as pd
+from pandas import DataFrame
+import matplotlib.pylab as plt
+import os
+from glob import glob
+
+INPUT_PATH = '../input' #Get the paths to the input data
+DATA_PATH = INPUT_PATH
+
+TRAIN_DATA = os.path.join(DATA_PATH, "train")
+TRAIN_MASKS_DATA = os.path.join(DATA_PATH, "train_masks")
+#TEST_DATA = os.path.join(DATA_PATH, "test") #comment out until actual testing
```

Comment:
Why does this need to be commented out?

Reply:
I commented it out because I didn't have enough storage on my vm to download the test images at the time.